src/components/molecules/DateInput/MonthSwiper/MonthSwiper.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { months } from '../DateInput.model';
+import { IMonthSwiper } from './MonthSwiper.model';
+import {
+  StyledArrowButton,
+  StyledArrowIcon,
+  StyledMonthSwiper,
+  StyledMonthText,
+} from './MonthSwiper.styled';
+
+const MonthSwiper = ({
+  isRightButtonDisabled,
+  isLeftButtonDisabled,
+  setIsAnimationStart,
+  animationClassName,
+  isAnimationStart,
+  handleRightArrow,
+  handleLeftArrow,
+  monthNumber,
+}: IMonthSwiper): JSX.Element => {
+  return (
+    <StyledMonthSwiper>
+      <StyledArrowButton disabled={isLeftButtonDisabled} onClick={handleLeftArrow}>
+        <StyledArrowIcon position="left" />
+      </StyledArrowButton>
+      <StyledMonthText
+        onAnimationEnd={() => setIsAnimationStart(false)}
```

Comment:
I cannot see any difference with this props or without. I just removed it and see the same behaviour